app/services/void_participant_declaration.rb

```diff
@@ -31,6 +32,7 @@ def make_voided
     raise Api::Errors::InvalidTransitionError, I18n.t(:declaration_not_voidable) unless participant_declaration.voidable?
 
     ApplicationRecord.transaction do
+      track_voiding_user
```

Comment:
I'm in two minds around where to have this, maybe we only have it in the void declaration service and instead of implementing it twice in each service we check if the clawback or voided was successful and track the voided user here once for both?

Reply:
I opted to put it in both services to make it more robust, in case `Finance::ClawbackDeclaration` was called independently (we do this in `Importers::Clawbacks`, for example). It's less likely this is going to be called manually from the rails console I guess, so happy to limit it to just the `VoidParticipantDeclaration` service if you feel that would be better?

Reply:
let's keep it